subsys/bluetooth/mesh/Kconfig

```diff
@@ -13,92 +13,168 @@ menuconfig BT_MESH
 
 if BT_MESH
 
-choice BT_MESH_CRYPTO_LIB
-	prompt "Crypto library selection for mesh security"
-	default BT_MESH_USES_TFM_PSA if BUILD_WITH_TFM
-	default BT_MESH_USES_TINYCRYPT
+###################################################################################################
+# Invisible options
+###################################################################################################
 
-config BT_MESH_USES_TINYCRYPT
-	bool "Use TinyCrypt"
-	select TINYCRYPT
-	select TINYCRYPT_AES
-	select TINYCRYPT_AES_CMAC
-	select TINYCRYPT_ECC_DH
-	select TINYCRYPT_SHA256
-	select TINYCRYPT_SHA256_HMAC
-	select BT_HOST_CCM
+# Virtual option enabled whenever any Proxy protocol is needed
```

Comment:
Hm... Mesh has a single Proxy protocol why `any`?
If you assumed server\client separation or pb-gatt, please, add clarification.